cypress/integration/smoke/balances.spec.js

```diff
@@ -0,0 +1,104 @@
+const assetsTableContainer = '.MuiTableContainer-root'
+const balanceRowTestId = '[data-testid=balance-row]'
+const receiveModalClass = '.receive-modal'
+
+const SAFE_TEST = '0x8675B754342754A30A2AeF474D114d8460bca19b'
+const ASSETS_LENGTH = 26
+
+describe('Assets > Coins', () => {
+  const number = 123456.789
+  const localeNumber = number.toLocaleString()
+  // Find localeNumber thousand separator and decimal separator
+  const [thousandSeparator, decimalSeparator] = localeNumber.match(/[^0-9]/g)
+
+  // Fiat balance regex
+  const fiatRegex = new RegExp(`([0-9]{1,3}${thousandSeparator})*[0-9]{1,3}${decimalSeparator}[0-9]{2}`)
+  // Token balance regex
+  const tokenRegex = new RegExp(`([0-9]{1,3}${thousandSeparator})*[0-9]{1,3}${decimalSeparator}[0-9]{5}`)
```

Comment:
IMHO that's too complicated to tests. You know the locale the tests are running, right? Just make literal assertions based on that.

Reply:
I thought about that approach initially but the values in fiat keep changing which made me discard literal assertions. Then I thought that the Token balances could also change due to some activity in the Safe so I took the regex approach considering the locale separators.

Reply:
I see. But still, just use `,` and `.` literally, no need to be super flexible.